src/components/AudioInfo.vue

```diff
@@ -1063,9 +1061,7 @@ const handleChangeSpeedScaleInput = (inputValue: string | number | null) => {
   });
 };
 
-const handleChangePitchScaleInput = (inputValue: string | number | null) => {
-  if (typeof inputValue !== "string")
-    throw new Error("typeof inputValue !== 'string'");
+const handleChangePitchScaleInput = (inputValue: string) => {
```

Comment:
VSCodeの拡張機能から見ると`q-input`での`@change`の定義が存在せずanyになっていることが少し気になります。

何かしらの理由で引数が変わってしまったことに気づけるようにするならこうするべき?
```suggestion
const handleChangePitchScaleInput = (inputValue: unknown) => {
  if (typeof inputValue !== "string")
    throw new Error("typeof inputValue !== 'string'");
```

Reply:
良いと思います！！

Reply:
他の部分含めて修正しました